linux_os/guide/system/software/disk_partitioning/partition_for_srv/rule.yml

```diff
@@ -17,8 +17,6 @@ rationale: |-
 
 severity: unknown
 
-platform: machine
```

Comment:
Torn. Worth further discussion. We do not *want* containers to have partitions but there is legitimacy in them too.

Reply:
I agree. In theory, we could check partition configuration in an offline scan, but

1. That would go against container practices, and
2. Only system setting check would be applicable, not the runtime one.